.github/workflows/test-recent-optimism-block.yml

```diff
@@ -14,13 +14,7 @@ jobs:
         with:
           ref: feat/multichain
 
-      - name: Install Rust (stable)
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          override: true
-
-      - uses: Swatinem/rust-cache@v2
+      - uses: ./.github/actions/setup-rust
```

Comment:
`main` was already merged to `feat/multichain`, so this should be safe to do, i.e. the action should already be there.